src/firefly/js/ui/ListBoxInputField.jsx

```diff
@@ -32,7 +32,9 @@ export function ListBoxInputFieldView({value:fieldValue='', onChange, fieldKey,
                 <Tooltip {...{title:tooltip, placement:'top', ...slotProps?.tooltip}}>
                     <Select {...{name: fieldKey, multiple, onChange, placeholder, renderValue, startDecorator,
                         disabled: readonly, size, value: multiple ? vAry : fieldValue,
-                        ...slotProps?.input}}>
+                        slotProps: {...slotProps?.input,
+                                listbox: {...(slotProps?.listbox ?? {}),}}
+                    }}>
```

Comment:
`input` slotprops is taking the props of `Select` component. So consumer  component can directly pass the `listbox` key in the object, no need to add another slotprop here.